tests/regression/test_mesh_generation.py

```diff
@@ -435,10 +435,3 @@ def test_changing_default_reorder_works(reorder):
     finally:
         parameters["reorder_meshes"] = old_reorder
 
-
-@pytest.mark.parametrize("kind, num_cells",
-                         [("default", 6), ("crossed", 5)])
```

Comment:
Please only remove the crossed case. Default should still be fine.

Reply:
Yes, this is now edited in the newest commit.